CIP-XXXX/README.md

```diff
@@ -0,0 +1,139 @@
+---
+CIP: ????
+Title: Cardano Governance Identifier
+Status: Proposed
+Category: Wallets
```

Comment:
```suggestion
Category: Tools
```
I am sure `Wallets` will be a huge part of this but the _Motivation_ also suggests other tools will process these IDs.  But @ashisherc @Ryun1 if it can be shown that this CIP is only for how wallets will generate & interpret the keys then please let's establish this here and in the CIP.

Reply:
Tools is the right category